action.yml

```diff
@@ -0,0 +1,101 @@
+name: "Sourcery GitHub Action"
+
+branding:
+  icon: 'box'
+  color: 'orange'
+
+description: "A GitHub Action for running Sourcery."
+
+inputs:
+  token:
+    description: "The Sourcery token for logging in."
+    required: true
+  target:
+    description: "File(s) or directory(ies) to review."
+    required: false
+    default: "."
+  version:
+    description: "Sourcery CLI version, e.g. '0.12.10'. If empty, default to the latest version."
+    required: false
+    default: ""
+  verbose:
+    description: "Verbose output with explanation and code snippets. Either 'true' or 'false'."
```

Comment:
🎯 suggestion: Can we define `type: boolean` and `default: true`?
Similar to described here:
https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/
That would save us some manual validation.

Reply:
I wish we could! But the link you posted refers to manual workflows, not to composite actions as is the case here. [The official docs for GitHub Action inputs do not describe the `type` field](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs), which is a shame. I did not test if such a `type` field magically works, but I prefer not to rely on it since it would be an undocumented feature.